lib/jsdom/living/nodes/SVGElement.idl

```diff
@@ -0,0 +1,3 @@
+interface SVGElement : HTMLElement {}; 
+
+// TODO: It should be Element + https://svgwg.org/svg2-draft/types.html#InterfaceSVGElement
```

Comment:
Why can't you just make it Element for now? Why make it inherit from HTMLElement? I don't think we can merge this with HTMLElement...